source/docs/yearly-overview/yearly-changelog.rst

```diff
@@ -5,10 +5,13 @@ New for 2023
 
 A number of improvements have been made to FRC\ |reg| Control System software for 2023. This article will describe and provide a brief overview of the new changes and features as well as a more complete changelog for Java/C++ WPILib changes. This document only includes the most relevant changes for end users, the full list of changes can be viewed on the various `WPILib <https://github.com/wpilibsuite/>`__ GitHub repositories.
 
-Due to internal GradleRIO changes, it is necessary to update previous years projects. After :doc:`Installing WPILib for 2023 </docs/zero-to-robot/step-2/wpilib-setup>`, any 2022 projects must be :doc:`imported </docs/software/vscode-overview/importing-gradle-project>` to be compatible.
-
 It's recommended to also review the list of :doc:`known issues <known-issues>`.
 
+Importing Previous Year's Projects
```

Comment:
I think the apostrophe is in the wrong place here. Regardless,  maybe rephrase to "importing projects from previous years"? Applies to the paragraph itself as well.